code/datums/ert/outsider.dm

```diff
@@ -12,6 +12,7 @@
 	name = "Interstellar Aid Corps"
 	chance = 15
 	spawner = /datum/ghostspawner/human/ert/iac
+	equipment_map = new /datum/map_template/distress_iac
```

Comment:
Creating the equip map shouldn't be done here, really. Instead it should either be done when the teams are loaded or when the equipment map is.

Reply:
Handled.